app_grim_reefer/src/config.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
```

Comment:
Prefer header guards

Reply:
Sure. I'm curious what is the reasoning for going the header guard route. Is it consistency with the rest of zephyr, like compiler compatibility, or something else fun?

Reply:
Just consistency throughout the project for now and being prepared for random bs. I don't think we have to worry too much about compiler compatibility. Hopefully never have to use some obscure compiler for some chip architecture. Also, in case of random Zephyr bs with files that screws with pragma. Just better peace of mind

Reply:
Not against pragma once, but that may have to be a future decision. Don't want to deal with the headache of random build issues right before a launch.